src/renderer/scss/component/_header.scss

```diff
@@ -1,17 +1,37 @@
 .header {
+  position: fixed;
```

Comment:
Now that the header is fixed, should the address bar stretch to take the remaining space available?

Reply:
yes